2022-03-27 01:23:48; IP: UNKNOWN;
URL: /api/account/registration;
Exception: 2 call_user_func_array() expects parameter 1 to be a valid callback, class '\App\Middleware\Role' not found D:\Projects\scrim\OldEngine\Http\Middleware\Middleware.php 41 in D:\Projects\scrim\OldEngine\ErrorHandler\Production\ErrorHandler.php:34
Stack trace:
#0 D:\Projects\scrim\OldEngine\ErrorHandler\Production\ErrorHandler.php(53): OldEngine\ErrorHandler\Production\ErrorHandler::showException(500, '2 call_user_fun...')
#1 [internal function]: OldEngine\ErrorHandler\Production\ErrorHandler->exception_handler(2, 'call_user_func_...', 'D:\\Projects\\scr...', 41, Array)
#2 D:\Projects\scrim\OldEngine\Http\Middleware\Middleware.php(41): call_user_func_array('\\App\\Middleware...', Array)
#3 D:\Projects\scrim\OldEngine\Routes\Route.php(410): OldEngine\Http\Middleware\Middleware::execute(Array, Object(Closure))
#4 D:\Projects\scrim\OldEngine\Routes\Route.php(395): OldEngine\Routes\Route->runRouteLogic('AccountControll...', Array)
#5 D:\Projects\scrim\OldEngine\Routes\Route.php(201): OldEngine\Routes\Route->_executeRoute('/api/account/re...', 'AccountControll...', Array)
#6 D:\Projects\scrim\app\routes\route.php(30): OldEngine\Routes\Route->post('/account/regist...', 'AccountControll...', Array)
#7 D:\Projects\scrim\OldEngine\Routes\Route.php(82): include_once('D:\\Projects\\scr...')
#8 D:\Projects\scrim\OldEngine\Application.php(68): OldEngine\Routes\Route->run()
#9 D:\Projects\scrim\public\index.php(11): OldEngine\Application->run()
#10 {main}

